src/components/EditableText.tsx

```diff
@@ -4,22 +4,26 @@ interface EditableProps {
   isEditing: boolean
   value: string
   onChangeHandler: React.ChangeEventHandler<HTMLInputElement>
-  onStopEditing: React.FocusEventHandler<HTMLInputElement>
+  onStopEditing: () => void
   onStartEditing: React.UIEventHandler<HTMLInputElement>
+  opts?: { isLarge: boolean }
 }
 export class EditableText extends React.Component<EditableProps> {
   render() {
+    const isLarge = this.props.opts && this.props.opts.isLarge
+
     return (
       <div className={this.controlClasses()}>
         <input
-          className={this.inputClasses()}
+          style={{textTransform: 'initial'}}
+          className={this.inputClasses() + (isLarge ? ' is-large' : '')}
           value={this.props.value}
           onChange={this.props.onChangeHandler}
           onBlur={this.props.onStopEditing}
           onClick={this.onClick}
         />
         {!this.props.isEditing && (
-          <span className="icon is-right">
+          <span className="icon is-right" style={isLarge ? {fontSize: '24px'} : {}}>
```

Comment:
I would try adding the `is-medium` or `is-large` class onto the span and/or `fa-lg` class onto the icon itself, in place of changing the font size directly, eg
```typescript
  <span className="icon is-right is-large" >
    <i className="fa fa-lg fa-edit" />
  </span>
```